modules/atomic-widgets/widgets/atomic-svg.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace Elementor\Modules\AtomicWidgets\Widgets;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Classes_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Primitives\String_Prop_Type;
+use Elementor\Modules\AtomicWidgets\Base\Atomic_Widget_Base;
+use Elementor\Core\Utils\Svg\Svg_Sanitizer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Atomic_Svg extends Atomic_Widget_Base {
+	public function get_name() {
+		return 'a-svg';
+	}
+
+	public function get_title() {
+		return esc_html__( 'Atomic SVG', 'elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-shape';
+	}
+
+	protected function render() {
+		$settings = $this->get_atomic_settings();
+
+		if ( ! isset( $settings['svg'] ) ) {
+			return;
+		}
+
+		$svg = $settings['svg'];
+
+		$svg = new \WP_HTML_Tag_Processor( $svg );
+
+		if ( ! $svg->next_tag() ) {
+			return;
+		}
+
+		$this->set_svg_attributes( $svg, $settings );
+		$this->set_svg_stroke_attributes( $svg, $settings );
+
+		$svg->add_class( $settings['classes'] ?? '' );
+
+		$valid_svg = ( new Svg_Sanitizer() )->sanitize( $svg->get_updated_html() );
+
+		//we need this line in order to render the svg, otherwise it will be rendered as a string
+		echo ( false === $valid_svg ) ? '' : $valid_svg; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	private function set_svg_attributes( $svg, $settings ) {
+		$svg->set_attribute( 'fill', $settings['color'] ?? 'currentColor' );
+		$svg->set_attribute( 'width', $settings['width'] ?? '100%' );
+		$svg->set_attribute( 'height', $settings['height'] ?? '100%' );
+		$svg->set_attribute( 'min-width', $settings['min-width'] ?? '100%' );
+		$svg->set_attribute( 'min-height', $settings['min-height'] ?? '100%' );
+		$svg->set_attribute( 'max-width', $settings['max-width'] ?? '100%' );
+		$svg->set_attribute( 'max-height', $settings['max-height'] ?? '100%' );
+	}
+
+	private function set_svg_stroke_attributes( $svg, $settings ) {
+		if ( isset( $settings['-webkit-text-stroke'] ) ) {
+			$stroke_parts = explode( ' ', $settings['-webkit-text-stroke'] );
+			if ( count( $stroke_parts ) === 2 ) {
+				$svg->set_attribute( 'stroke-width', $stroke_parts[0] ?? '0px' );
+				$svg->set_attribute( 'stroke', $stroke_parts[1] ?? 'currentColor' );
+			}
+		}
```

Comment:
```suggestion
		if ( isset( $settings['-webkit-text-stroke'] ) ) {
			$stroke_parts = explode( ' ', $settings['-webkit-text-stroke'] );
			if ( count( $stroke_parts ) === 2 ) {
				$svg->set_attribute( 'stroke-width', $stroke_parts[0] ?? '0px' );
				$svg->set_attribute( 'stroke', $stroke_parts[1] ?? 'currentColor' );
			}
		}
```
		if (! isset( $settings['-webkit-text-stroke'] ) ) {
			return ;
		}

		$stroke_parts = explode( ' ', $settings['-webkit-text-stroke'] );

		if ( ! count( $stroke_parts ) === 2 ) {
			return ;
		}

		$svg->set_attribute( 'stroke-width', $stroke_parts[0] ?? '0px' );
		$svg->set_attribute( 'stroke', $stroke_parts[1] ?? 'currentColor' );